src/mlnet/Telemetry/Events/SystemInfoEvent.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.CLI.Telemetry.Events
+{
+    /// <summary>
+    /// System info telemetry event.
+    /// </summary>
+    internal class SystemInfoEvent
```

Comment:
why is this not a part of the initial invocation event?

Reply:
@justinormont pointed out that having a separate log statement here could be preferable, as not to clog up the "start command" event telemetry as the amount of system information grows

Reply:
I realize that this is also laid out more here https://github.com/dotnet/machinelearning-automl/issues/417